rc/gas.kak

```diff
@@ -0,0 +1,61 @@
+# Detection
+# ---------
+hook global BufSetOption mimetype=text/x-python %{
+    set buffer filetype gas
+}
+
+hook global BufCreate .*\.(s|S|asm)$ %{
+    set buffer filetype gas
```

Comment:
This is could be a solution, except not all `.s` files use follow the GAS syntax. Some trickery might be needed here, maybe a quick grep into the buffer to find unique keywords that specific to this syntax, to be sure of the type of file you're dealing with ?